Demo/Common/Integrations/ComponentIntegrationController.swift

```diff
@@ -0,0 +1,174 @@
+//
+//  ComponentIntegrationController.swift
+//  Adyen
+//
+//  Created by Vladimir Abramichev on 05/02/2021.
+//  Copyright © 2021 Adyen. All rights reserved.
+//
```

Comment:
I thought we don't use this kind of header anymore.

Reply:
nope, cleaned